repo/blob/s3/s3_storage_test.go

```diff
@@ -111,24 +205,24 @@ func testStorage(t *testing.T, accessID, secretKey, sessionToken string) {
 	}
 }
 
-func createBucket(t *testing.T) {
-	minioClient, err := minio.New(endpoint, accessKeyID, secretAccessKey, useSSL)
+func createMinioBucket(t *testing.T, bucketName string) {
```

Comment:
It makes sense to generalize `createBucket` to work for (generic/AWS) S3 as well. If you pass `Options{...}` as a parameter and a `useSSL` flag, then you have everything you need, no?

Reply:
Need to check if minioclient works with aws otherwise we'll need to have separate handling for creating AWS bucket

Reply:
I gather it works given the changes.